index.html

```diff
@@ -11,6 +11,6 @@
     <h1>Hello world</h1>
     A
     B
-    
+    C
```

Comment:
C should be small case, you have added Big C.

Reply:
Ok i am going to change it.

Reply:
I have made changes @Bashaboina please check it.

Reply:
I have made changes @Bashaboina please check it.

Reply:
Looks good for me @Himabindujunnuthula